module/VuFind/tests/unit-tests/src/VuFindTest/Role/PermissionProvider/UserTest.php

```diff
@@ -150,17 +151,17 @@ protected function getMockAuthorizationService()
     /**
      * Get a mock user object
      *
-     * @return \VuFind\Db\Row\User
+     * @return \VuFind\Db\Entity\UserEntityInterface
```

Comment:
Since there is a `use` statement for this class, you can simplify the annotation here:

```suggestion
     * @return UserEntityInterface
```